charts/argo-workflows/templates/controller/workflow-role.yaml

```diff
@@ -24,6 +24,12 @@ rules:
     verbs:
       - get
       - watch
+  - apiGroups:
+      - argoproj.io
+    resources:
+      - workflowtaskresults
+    verbs:
+      - create
```

Comment:
Is it sufficient with only this verb? Have u tested it?

Reply:
I have the following error in my logs which suggests it might also need `list` and `watch` too...

```
W0508 04:13:37.009819       1 reflector.go:324] pkg/mod/k8s.io/client-go@v0.23.3/tools/cache/reflector.go:167: failed to list *v1alpha1.WorkflowTaskResult: workflowtaskresults.argoproj.io is forbidden: User "system:serviceaccount:argo-workflows:argo-workflows-workflow-controller" cannot list resource "workflowtaskresults" in API group "argoproj.io" at the cluster scope
E0508 04:13:37.009837       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.3/tools/cache/reflector.go:167: Failed to watch *v1alpha1.WorkflowTaskResult: failed to list *v1alpha1.WorkflowTaskResult: workflowtaskresults.argoproj.io is forbidden: User "system:serviceaccount:argo-workflows:argo-workflows-workflow-controller" cannot list resource "workflowtaskresults" in API group "argoproj.io" at the cluster scope
```

Reply:
Thanks. Fancy doing a PR?

Reply:
Wasn't aware of this before my MR, but according to this documentation [workflowtaskresults](https://argoproj.github.io/argo-workflows/workflow-rbac/#workflow-rbac).  The verbs _create_ and _patch_ should be sufficient as far as I can tell for most executor types. Haven't noticed the missing list/watch permissions for my use case.

Reply:
Seems to have been [addressed already](https://github.com/argoproj/argo-helm/commit/5f33394042e8ea27d0fe95187534a410a35541ee). Thanks!